bst.py

```diff
@@ -0,0 +1,126 @@
+# http://blog.shayanjaved.com/2012/01/14/binary-search-tree-in-python/
+
+# http://interactivepython.org/XSKWZ/LpOMZ/courselib/static/pythonds/Trees/bst.html
+import timeit
+import random
+
+
+class BSTNode(object):
+    '''Instantiate Node and add helper functions'''
+    def __init__(self, value, parent=None, left_child=None, right_child=None):
+        self.value = value
+        self.parent = parent
+        self.left = left_child
+        self.right = right_child
+
+    def is_root(self):
+        '''Helper function for root node'''
+        return not self.parent
+
+    def is_leaf(self):
+        ''''Helper function for acknowledging leaf'''
+        return not (self.right_child or self.left_child)
+
+
+class BST(object):
+    '''Instantiate binary search tree'''
+    def __init__(self, values=None):
```

Comment:
What would happen if the user passed values into this init function?  Anything?  Why is the argument there?